integrations-and-supported-tools/mosaicml-composer/scripts/Neptune_MosaicML_Composer.py

```diff
@@ -3,18 +3,21 @@
 # Script adapted from https://docs.mosaicml.com/projects/composer/en/latest/examples/getting_started.html
 # Date accessed: 2024-02-06
 
+## Import libraries
 import composer
 import torch
 import torch.nn as nn
 import torch.nn.functional as F
 import torch.utils.data
 from composer.algorithms import LabelSmoothing, ProgressiveResizing
-from composer.callbacks import ImageVisualizer
+from composer.callbacks import *
 from composer.loggers import NeptuneLogger
 from composer.models import ComposerClassifier
+from neptune import ANONYMOUS_API_TOKEN  # Needed only for ANONYMOUS runs
```

Comment:
Maybe this angle is better

```suggestion
from neptune import ANONYMOUS_API_TOKEN  # Not needed if you use your own Neptune credentials
```